docker-compose.yml

```diff
@@ -5,7 +5,6 @@ services:
     image: tabulario/spark-iceberg
     depends_on:
       - postgres
-    container_name: spark-iceberg
```

Comment:
What's the reasoning for removing the `container_name`?

Reply:
I was running into issues when I run the docker-compose from different directories from container name conflicts. It was enough enough to kill and rm those containers but I realized they aren't actually required so I removed it to cause less potential friction for users.